directory/src/App.js

```diff
@@ -109,6 +129,21 @@ class SearchBar extends PureComponent {
               placeholder="Search for an icon..."
             />
           </form>
+          <div className="Search-Select-Families">
+            {Object.keys(IconFamilies).map(family => (
```

Comment:
Filter out `FontAwesome5Brands` from the keys here 🙂

Reply:
I am not sure to understand fully the impact. If we filter out `FontAwesome5Brands` should we handle a specific case when `FontAwesome5` is selected to also include iocns from `FontAwesome5Brands` ?

Reply:
Sorry, maybe I should have been more clear 🙂 Font Awesome 5 has a fallback and will include the brand icons either way but if you select only the brands family you'll see a lot of squares because the fallback is not implemented that way. So it would just be better to just include only the main family 🙂

Reply:
I add the filter like you ask and 2 buttons to Select all and Select none.